lib/furi/uri.rb

```diff
@@ -589,7 +589,7 @@ def mailto?
 
     def encoded_anchor
       return "" unless anchor
-      "#" + ::URI.encode(anchor)
+      "#" + ::URI::DEFAULT_PARSER.escape(anchor)
```

Comment:
`URI.escape` and `URI.unescape` have been [considered obsolete since 2009](https://github.com/ruby/ruby/commit/238b979f1789f95262a267d8df6239806f2859cc) and were removed in Ruby 3.0.
`URI::DEFAULT_PARSER` was not removed though and has the same behavior.